data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+			`	As you follow them, Adrauni begins to speak while holding out one arm. "If you may, as humans say, humor me, what do you suspect was the result of my negotiations with the Gegno?"`
+			choice
+				`	"That they will accept me?"`
+				`	"That they want nothing to do with me?"`
+					goto "deny me"
+				`	"I don't know."`
+					goto "unsure"
+
+			action
+				set "gegno word: accept"
+			`	After a brief delay, Adrauni turns to the Gegno, mutters something in their language, then responds to you. "I see. And what do you propose is the reasoning for that?"`
+			choice
+				`	"I've helped them before, and I could help them now."`
+				`	"I figured you could convince them."`
+					goto "convince"
+				`	"I don't know."`
+					goto "unsure"
+
+			`	Adrauni ponders for a few moments as you continue walking through the impeccable interior of the ringworld. "That is a rather human response. However, do not conflate your value in assistance with theirs. Even should you provide beneficial assistance to the Gegno, a beneficiary response in return is not necessitated. You have heeded my warnings at this juncture, yet you simply are doing what the Gegno desired from the beginning, and that is to be left alone."`
+			choice
+				`	"Maybe it's better I do leave them alone."`
+					goto "better"
+				`	"There must be something I can do."`
+					goto "understand"
+
+			label "convince"
+			`	Adrauni gives you a slight glare while continuing through the ringworld. "Do you not recall my prior mention of us Quarg having attempted such desired persuasions? It is not within our ability to convince the Gegno of anything. They are greatly devoted to their own, and will go beyond any boundary to oppose counsel, even by us Quarg."`
+			choice
+				`	"Maybe it's better I do leave them alone."`
+					goto "better"
+				`	"There must be something I can do."`
+					goto "understand"
+
+			label "deny me"
+			action
+				set "gegno word: reject"
+			`	After a brief delay, Adrauni turns to the Gegno, mutters something in their language, then responds to you. "I see. And what do you propose is the reasoning for that?"`
+			choice
+				`	"The way they act around me."`
+				`	"From what you've told me before."`
+					goto "before"
+				`	"I don't know."`
+					goto "unsure"
+
+			`	"Indeed, the Gegno are wary of you, who lacks caution." A few seconds, and Adrauni gives a strange look towards the Gegno. You aren't sure what it's thinking. The Gegno asks something in their language, but receives no response. A few more moments go by before Adrauni speaks again. "Despite that, you have found yourself in precarious situations we have not. We as a species embraced contact with the Gegno, however, you remain the only human here. Perhaps therein lies a notable implication of distinction."`
```

Comment:
```suggestion
			`	"Indeed, the Gegno are wary of you, who lacks caution." A few seconds pass, and Adrauni gives a strange look towards the Gegno. You aren't sure what it's thinking. The Gegno asks something in their language, but receives no response. A few more moments go by before Adrauni speaks again. "Despite that, you have found yourself in precarious situations we have not. We as a species embraced contact with the Gegno, however, you remain the only human here. Perhaps therein lies a notable implication of distinction."`
```
Does First Last already have an understanding of when a Gegno speaker is _asking_ something?
If not, maybe a blurb about how the Gegno seems to look at the Quarg expectantly but then give up waiting for an answer?